src/content/guides/asset-management.md

```diff
@@ -336,37 +297,47 @@ With the loader configured and fonts in place, you can incorporate them via an `
 __src/style.css__
 
 ``` diff
-+ @font-face {
-+   font-family: 'MyFont';
-+   src:  url('./my-font.woff2') format('woff2'),
-+         url('./my-font.woff') format('woff');
-+   font-weight: 600;
-+   font-style: normal;
-+ }
-
-  .hello {
-    color: red;
-+   font-family: 'MyFont';
-    background: url('./icon.png');
-  }
++@font-face {
++  font-family: 'GoogleFont';
```

Comment:
Why this change? People may get confused and will start downloading GoogleFonts instead of using their CDN.

I think this example is targeted for use cases when your font is not available anywhere and you have to ship ip with your project.